site/zh/blog/new-in-devtools-95/index.md

```diff
@@ -29,53 +29,68 @@ tags:
 
 
 <!-- ## New CSS length authoring tools {: #length } -->
+## 新的 CSS 长度编辑工具 {: #length }
 
 <!-- DevTools added an easier yet flexible way to update lengths in CSS! -->
+DevTools 添加了一种更简单且灵活的方式来更新 CSS 长度！ 
 
 <!-- In the **Styles** pane, look for any CSS property with length (e.g. `height`, `padding`). -->
+在“样式”边栏中，找到任意一个含有长度的 CSS 属性。
```

Comment:
> In the **Styles** pane, look for any CSS property with length (e.g. `height`, `padding`)

The current translation of the above sentence is still in incomplete. Namely, you have ignored some content such as **(e.g. `height`, `padding`)**.